scripts/docstrings_checker.py

```diff
@@ -214,107 +212,3 @@ class GoogleDocstring(_check_docs_utils.GoogleDocstring):
     """.format(
         type=re_multiple_type,
     ), flags=re.X | re.S | re.M)
-
-
-class ASTDocStringChecker:
```

Comment:
WHy are we removing this checker?

Reply:
@DubeySandeep This class ASTDocStringChecker is redundant according to:-https://github.com/oppia/oppia/pull/14073#discussion_r736863243
PTAL!